src/calendar/index.tsx

```diff
@@ -268,11 +269,11 @@ const Calendar = (props: CalendarProps) => {
     const ref = customHeader ? undefined : header;
     const CustomHeader = customHeader;
     const HeaderComponent = customHeader ? CustomHeader : CalendarHeader;
-    
+
     return (
       <HeaderComponent
         {...headerProps}
-        testID={testID}
+        testID={`${testID}.header`}
```

Comment:
This is what might collide with the CalndarList's static header (comment above)

Reply:
I fixed the static header id, here we'll keep it as header

Reply:
Yes. That's what I meant. Thanks